brainstormforce/ultimate-addons-for-gutenberg
# Improvement: Lottie - Updated React Lottie Library.

### Description

- Removed old `react-lottie` dependency and code.
- Updated `@lottiefiles/react-lottie-player`

### Types of changes




- Improvement

### How has this been tested?

- Drag and drop lottie block
- Check all settings

### Checklist:
